lib/generators/blacklight/solr4_generator.rb

```diff
@@ -10,10 +13,8 @@ class Solr4Generator < Rails::Generators::Base
     EOS
 
     def install_jettywrapper
-      return unless options[:jettywrapper]
```

Comment:
It's possible to also get here from:

https://github.com/projectblacklight/blacklight/blob/master/lib/generators/blacklight/install_generator.rb#L35

Seems like we ought to pass the generator options through, maybe?